packages/blocks/src/api/categories.ts

```diff
@@ -57,7 +62,7 @@ export function getCategories() {
  * };
  * ```
  */
-export function setCategories( categories ) {
+export function setCategories( categories: readonly WPBlockCategory[] ): void {
```

Comment:
Do we need the `void` return type explicitly here? It's quite easy for TypeScript to infer it, so that we can avoid some noise.